### ISSUE TITLE
The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' is not announced NVDA

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7250]
**Test Environment**:
OS: Windows_11
Version: 22H2
OS Build: 22598.200
Browser: Version 104.0.1293.70 (Official Build) (64-bit)
URL: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering

**Repro Steps**:
1. Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering
2. Angular excel style filtering page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the sorting(ascending/descending) information is announced.

**Actual Behavior:**
The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' is not announced NVDA.

**Expected Behavior:**
The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' should be announced NVDA.

**Attachments:**
[MAS1.3.1_sorting info not announced.zip](https://github.com/angular-ui/ui-grid/files/9557074/MAS1.3.1_sorting.info.not.announced.zip)

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
